homeworks/olha.tymchyshyn_olyatmchshn/PokemonProject/css/login.css

```diff
@@ -0,0 +1,192 @@
+html {
+  height: 100%;
+  width: 100%;
+}
+
+/* HEADER STYLES */
+
+.page-header {
+  background-color: #333;
+  width: 100%;
+}
+
+.page-header-container {
+  display: flex;
+  justify-content: space-between;
+  max-width: 100%;
+  height: 80px;
+  align-items: center;
+  padding-right: 53px;
+  padding-left: 53px;
+}
+
+.logo {
+  width: 190px;
+  height: 29.945px;
+}
+
+.nav-menu {
+  width: 260px;
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.nav-menu-item-link {
+  text-decoration: none;
+  color: #FFF;
+  font-family: Roboto;
+  font-size: 16px;
+  font-weight: 500;
+}
+
+.nav-menu-item-link:hover,
+.nav-menu-item-link:focus {
+  color: #F04934;
+}
+
+/* MAIN STYLES */
+
+.login-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+h1 {
+  color: #221F1F;
+  font-family: Roboto;
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 32px;
+  margin-top: 80px;
+  margin-bottom: 60px;
+}
+
+input[type="email"] {
+  border-radius: 6px;
+  border: 1px solid #A9A9A9;
+  width: 575px;
+  height: 70px;
+  display: flex;
+  flex-direction: column;
+  color: #A9A9A9;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 150%;
+  padding-left: 20px;
+  margin-top: 22px;
+  margin-bottom: 40px;
+}
+
+input[type="password"] {
+  border-radius: 6px;
+  border: 1px solid #A9A9A9;
+  width: 575px;
+  height: 70px;
+  display: flex;
+  flex-direction: column;
+  color: #A9A9A9;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 150%;
+  padding-left: 20px;
+  margin-top: 22px;
+  margin-bottom: 40px;
+}
+
+label {
+  width: 575px;
+  color: #221F1F;
+  font-family: Roboto;
+  font-size: 22px;
+  font-weight: 600;
+}
+
+.robotcheck {
+  display: block;
+  margin-bottom: 40px;
+}
+
+.robotcheck-label {
+  color: #221F1F;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 150%;
+  cursor: pointer;
+  position: relative;
+}
+
+input[type="checkbox"] {
+  padding: 3px;
+  margin-bottom: 40px;
+  cursor: pointer;
+  display: none;
```

Comment:
This hides not only native checkbox, but the validation mesage also. In result, a user could be confused, why they can't submit the form.
<img width="792" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/1ca7e184-c90e-4fc0-ac05-967243fffda7">